### ISSUE TITLE
Duplicate .drop-lounging CSS in skin.css

### ISSUE THREAD

**Issue by jlfranklin** [backdrop/backdrop-issues#4625]
The CSS that places Drop in the Powered-By footer block is in the `footer.css` file.  However, part of the CSS is also duplicated in `skin.css`.

This bit is in `skin.css`.  It's also in `component/footer.css`, minus the comments.
```css
.drop-lounging {
  /* To make Drop larger/smaller, adjust the width below. */
  width: 7em;
}

.drop-lounging:before {
  /* Uncomment below to remove Drop, or remove markup from block--system--powered-by.tpl.php */
  /* display: none; */
  background-image: url("../images/drop-lounging.png");
}
```

Backdrop 1.17.0

**Comment by herbdool** [comment]
@jenlampton was this duplication on purpose? I was wondering if it helps with creating a child theme or if it's just cruft.

**Comment by jenlampton** [comment]
I don't think it was intentional.

**Comment by indigoxela** [comment]
Another try to fix CSS stuff in core... :grin: 

In theory it *could* have been a try to make things easier for subthemes, but then again I see no real consistency regarding that in skin.css.

As @jenlampton seems to share the same impression, I'm marking this as RTBC.

**Comment by klonos** [comment]
> In theory it *could* have been a try to make things easier for subthemes...

Yup, that's the reason. People might get annoyed by the dragon being so prominent in their theme/design, so they might have looked for an easy way to make it smaller, or make it go away. The easy solution from them would be to remove the block entirely, but that would also make the "Powered by Backdrop CMS" text/link also go away, which is something we would like to have sites keep (for SEO/marketing purposes etc.). I think that this was an attempt to allow making the block more subtle, without having themers do a lot of work.

**Comment by indigoxela** [comment]
> People might get annoyed by the dragon being so prominent in their theme/design

Sure, but then it's much easier to replace the block with something custom. :wink: And this also doesn't explain why it's there twice.

**Comment by BWPanda** [comment]
I'm hesitant to merge this just yet... Are we sure there's no reason for this, and that removing that CSS won't adversely affect anything (especially existing sites)?

**Comment by quicksketch** [comment]
We changed the size of Drop in the footer in https://github.com/backdrop/backdrop-issues/issues/2182, in which we changed the width value in both CSS files. So it appears as though this duplication was indeed intentional.

I can't see how this would affect any existing sites.

I'm in favor of removing the duplicate lines, despite the original intention to make it easy for sub-themers to do something different. It's more likely that a site author would just disable the block like @indigoxela suggested.

So +1 from me. I'm curious if @stpaultim or @wesruv would be able to weigh in here.

**Comment by wesruv** [comment]
It was (kind of) intentional. 

My thought was for colors, size, and fonts (probably other small, not layout breaking things) that you'd only need to override skin.css

(This was before the color module.)

Last I touched basis, I was hit or miss on what values only showed up in skin.css, and the ones that I duplicated in component files.

At this moment, I prefer duplication, so that skin.css could go away and the component CSS could stand on it's own, and mashed sense in it's own.

I think either approach is fine, but I kept waffling on which approach I preferred when I made Basis, but never reconciled everything one way or the other.

The mix of approaches that ended up being the case aren't a technical problem (because they cascade/specificity is right) but it isn't amazing DX.
